## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 420345`
Commit: `0d81a6d4adf89d3bc7a5236deb9ceeba92aa157a`

---
### `aarch64-darwin`
<details>
  <summary>:x: 58 packages failed to build:</summary>
  <ul>
    <li>aider-chat (python312Packages.aider-chat)</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist (python312Packages.aider-chat.dist)</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 448 packages built:</summary>
  <ul>
    <li>alpaca</li>
    <li>chatblade</li>
    <li>chatblade.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>imaginer</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>litellm</li>
    <li>litellm.dist</li>
    <li>llm</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>openai (python313Packages.openai)</li>
    <li>openai.dist (python313Packages.openai.dist)</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.banks</li>
    <li>python312Packages.banks.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.fireworks-ai</li>
    <li>python312Packages.fireworks-ai.dist</li>
    <li>python312Packages.fnllm</li>
    <li>python312Packages.fnllm.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.llm</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.llm-cmd</li>
    <li>python312Packages.llm-cmd.dist</li>
    <li>python312Packages.llm-command-r</li>
    <li>python312Packages.llm-command-r.dist</li>
    <li>python312Packages.llm-deepseek</li>
    <li>python312Packages.llm-deepseek.dist</li>
    <li>python312Packages.llm-docs</li>
    <li>python312Packages.llm-docs.dist</li>
    <li>python312Packages.llm-echo</li>
    <li>python312Packages.llm-echo.dist</li>
    <li>python312Packages.llm-fragments-github</li>
    <li>python312Packages.llm-fragments-github.dist</li>
    <li>python312Packages.llm-fragments-pypi</li>
    <li>python312Packages.llm-fragments-pypi.dist</li>
    <li>python312Packages.llm-fragments-reader</li>
    <li>python312Packages.llm-fragments-reader.dist</li>
    <li>python312Packages.llm-fragments-symbex</li>
    <li>python312Packages.llm-fragments-symbex.dist</li>
    <li>python312Packages.llm-gemini</li>
    <li>python312Packages.llm-gemini.dist</li>
    <li>python312Packages.llm-gguf</li>
    <li>python312Packages.llm-gguf.dist</li>
    <li>python312Packages.llm-git</li>
    <li>python312Packages.llm-git.dist</li>
    <li>python312Packages.llm-grok</li>
    <li>python312Packages.llm-grok.dist</li>
    <li>python312Packages.llm-groq</li>
    <li>python312Packages.llm-groq.dist</li>
    <li>python312Packages.llm-hacker-news</li>
    <li>python312Packages.llm-hacker-news.dist</li>
    <li>python312Packages.llm-jq</li>
    <li>python312Packages.llm-jq.dist</li>
    <li>python312Packages.llm-llama-server</li>
    <li>python312Packages.llm-llama-server.dist</li>
    <li>python312Packages.llm-mistral</li>
    <li>python312Packages.llm-mistral.dist</li>
    <li>python312Packages.llm-ollama</li>
    <li>python312Packages.llm-ollama.dist</li>
    <li>python312Packages.llm-openai-plugin</li>
    <li>python312Packages.llm-openai-plugin.dist</li>
    <li>python312Packages.llm-openrouter</li>
    <li>python312Packages.llm-openrouter.dist</li>
    <li>python312Packages.llm-pdf-to-images</li>
    <li>python312Packages.llm-pdf-to-images.dist</li>
    <li>python312Packages.llm-perplexity</li>
    <li>python312Packages.llm-perplexity.dist</li>
    <li>python312Packages.llm-sentence-transformers</li>
    <li>python312Packages.llm-sentence-transformers.dist</li>
    <li>python312Packages.llm-templates-fabric</li>
    <li>python312Packages.llm-templates-fabric.dist</li>
    <li>python312Packages.llm-templates-github</li>
    <li>python312Packages.llm-templates-github.dist</li>
    <li>python312Packages.llm-tools-datasette</li>
    <li>python312Packages.llm-tools-datasette.dist</li>
    <li>python312Packages.llm-tools-quickjs</li>
    <li>python312Packages.llm-tools-quickjs.dist</li>
    <li>python312Packages.llm-tools-simpleeval</li>
    <li>python312Packages.llm-tools-simpleeval.dist</li>
    <li>python312Packages.llm-tools-sqlite</li>
    <li>python312Packages.llm-tools-sqlite.dist</li>
    <li>python312Packages.llm-venice</li>
    <li>python312Packages.llm-venice.dist</li>
    <li>python312Packages.llm-video-frames</li>
    <li>python312Packages.llm-video-frames.dist</li>
    <li>python312Packages.llm.dist</li>
    <li>python312Packages.llmx</li>
    <li>python312Packages.llmx.dist</li>
    <li>python312Packages.markitdown</li>
    <li>python312Packages.markitdown.dist</li>
    <li>python312Packages.openai</li>
    <li>python312Packages.openai-agents</li>
    <li>python312Packages.openai-agents.dist</li>
    <li>python312Packages.openai.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.smolagents</li>
    <li>python312Packages.smolagents.dist</li>
    <li>python312Packages.speechrecognition</li>
    <li>python312Packages.speechrecognition.dist</li>
    <li>python312Packages.sqlite-vec</li>
    <li>python312Packages.sqlite-vec.dist</li>
    <li>python313Packages.banks</li>
    <li>python313Packages.banks.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.fireworks-ai</li>
    <li>python313Packages.fireworks-ai.dist</li>
    <li>python313Packages.fnllm</li>
    <li>python313Packages.fnllm.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.litellm</li>
    <li>python313Packages.litellm.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.llm</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.llm-cmd</li>
    <li>python313Packages.llm-cmd.dist</li>
    <li>python313Packages.llm-command-r</li>
    <li>python313Packages.llm-command-r.dist</li>
    <li>python313Packages.llm-deepseek</li>
    <li>python313Packages.llm-deepseek.dist</li>
    <li>python313Packages.llm-docs</li>
    <li>python313Packages.llm-docs.dist</li>
    <li>python313Packages.llm-echo</li>
    <li>python313Packages.llm-echo.dist</li>
    <li>python313Packages.llm-fragments-github</li>
    <li>python313Packages.llm-fragments-github.dist</li>
    <li>python313Packages.llm-fragments-pypi</li>
    <li>python313Packages.llm-fragments-pypi.dist</li>
    <li>python313Packages.llm-fragments-reader</li>
    <li>python313Packages.llm-fragments-reader.dist</li>
    <li>python313Packages.llm-fragments-symbex</li>
    <li>python313Packages.llm-fragments-symbex.dist</li>
    <li>python313Packages.llm-gemini</li>
    <li>python313Packages.llm-gemini.dist</li>
    <li>python313Packages.llm-gguf</li>
    <li>python313Packages.llm-gguf.dist</li>
    <li>python313Packages.llm-git</li>
    <li>python313Packages.llm-git.dist</li>
    <li>python313Packages.llm-grok</li>
    <li>python313Packages.llm-grok.dist</li>
    <li>python313Packages.llm-groq</li>
    <li>python313Packages.llm-groq.dist</li>
    <li>python313Packages.llm-hacker-news</li>
    <li>python313Packages.llm-hacker-news.dist</li>
    <li>python313Packages.llm-jq</li>
    <li>python313Packages.llm-jq.dist</li>
    <li>python313Packages.llm-llama-server</li>
    <li>python313Packages.llm-llama-server.dist</li>
    <li>python313Packages.llm-mistral</li>
    <li>python313Packages.llm-mistral.dist</li>
    <li>python313Packages.llm-ollama</li>
    <li>python313Packages.llm-ollama.dist</li>
    <li>python313Packages.llm-openai-plugin</li>
    <li>python313Packages.llm-openai-plugin.dist</li>
    <li>python313Packages.llm-openrouter</li>
    <li>python313Packages.llm-openrouter.dist</li>
    <li>python313Packages.llm-pdf-to-images</li>
    <li>python313Packages.llm-pdf-to-images.dist</li>
    <li>python313Packages.llm-perplexity</li>
    <li>python313Packages.llm-perplexity.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.llm-templates-fabric</li>
    <li>python313Packages.llm-templates-fabric.dist</li>
    <li>python313Packages.llm-templates-github</li>
    <li>python313Packages.llm-templates-github.dist</li>
    <li>python313Packages.llm-tools-datasette</li>
    <li>python313Packages.llm-tools-datasette.dist</li>
    <li>python313Packages.llm-tools-quickjs</li>
    <li>python313Packages.llm-tools-quickjs.dist</li>
    <li>python313Packages.llm-tools-simpleeval</li>
    <li>python313Packages.llm-tools-simpleeval.dist</li>
    <li>python313Packages.llm-tools-sqlite</li>
    <li>python313Packages.llm-tools-sqlite.dist</li>
    <li>python313Packages.llm-venice</li>
    <li>python313Packages.llm-venice.dist</li>
    <li>python313Packages.llm-video-frames</li>
    <li>python313Packages.llm-video-frames.dist</li>
    <li>python313Packages.llm.dist</li>
    <li>python313Packages.llmx</li>
    <li>python313Packages.llmx.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.openai-agents</li>
    <li>python313Packages.openai-agents.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.sqlite-vec</li>
    <li>python313Packages.sqlite-vec.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>sploitscan</li>
    <li>sploitscan.dist</li>
    <li>textlsp</li>
    <li>textlsp.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>aider-chat</summary>
<pre>FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_lua - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_matlab - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml_interface - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_php - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_pony - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_properties - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_python - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_r - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_racket - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ruby - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_rust - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_scala - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_solidity - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_swift - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_tsx - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_typescript - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_udev - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_repo_map_sample_code_base - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
===== 42 failed, 417 passed, 1 skipped, 15 deselected in 267.32s (0:04:27) =====</pre>
</details>
<details>
<summary>aider-chat-full</summary>
<pre>FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_lua - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_matlab - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml_interface - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_php - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_pony - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_properties - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_python - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_r - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_racket - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ruby - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_rust - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_scala - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_solidity - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_swift - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_tsx - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_typescript - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_udev - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_repo_map_sample_code_base - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
===== 42 failed, 417 passed, 1 skipped, 15 deselected in 107.66s (0:01:47) =====</pre>
</details>
<details>
<summary>aider-chat-with-bedrock</summary>
<pre>FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_lua - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_matlab - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml_interface - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_php - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_pony - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_properties - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_python - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_r - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_racket - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ruby - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_rust - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_scala - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_solidity - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_swift - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_tsx - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_typescript - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_udev - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_repo_map_sample_code_base - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
===== 42 failed, 417 passed, 1 skipped, 15 deselected in 111.43s (0:01:51) =====</pre>
</details>
<details>
<summary>aider-chat-with-browser</summary>
<pre>FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_lua - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_matlab - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml_interface - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_php - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_pony - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_properties - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_python - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_r - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_racket - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ruby - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_rust - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_scala - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_solidity - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_swift - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_tsx - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_typescript - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_udev - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_repo_map_sample_code_base - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
===== 42 failed, 417 passed, 1 skipped, 15 deselected in 268.18s (0:04:28) =====</pre>
</details>
<details>
<summary>aider-chat-with-help</summary>
<pre>FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_lua - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_matlab - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml_interface - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_php - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_pony - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_properties - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_python - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_r - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_racket - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ruby - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_rust - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_scala - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_solidity - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_swift - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_tsx - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_typescript - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_udev - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_repo_map_sample_code_base - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
===== 42 failed, 417 passed, 1 skipped, 15 deselected in 279.89s (0:04:39) =====</pre>
</details>
<details>
<summary>aider-chat-with-playwright</summary>
<pre>FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_lua - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_matlab - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ocaml_interface - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_php - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_pony - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_properties - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_python - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_r - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_racket - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_ruby - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_rust - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_scala - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_solidity - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_swift - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_tsx - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_typescript - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_language_udev - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/basic/test_repomap.py::TestRepoMapAllLanguages::test_repo_map_sample_code_base - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
===== 42 failed, 417 passed, 1 skipped, 15 deselected in 267.47s (0:04:27) =====</pre>
</details>
<details>
<summary>python312Packages.langchain-openai</summary>
<pre>Name (time in ms)          Min                   Max                Mean              StdDev              Median                 IQR            Outliers      OPS            Rounds  Iterations
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_init_time          5.6126 (1.0)        149.2187 (1.80)      12.7971 (1.0)       19.3413 (1.61)       9.0936 (1.0)        4.4849 (1.0)           3;4  78.1424 (1.0)          67           1
test_init_time          5.7493 (1.02)        82.9804 (1.0)       13.2904 (1.04)      12.0050 (1.0)        9.5962 (1.06)       8.7812 (1.96)          9;8  75.2423 (0.96)        163           1
test_init_time        547.6522 (97.58)    1,823.2704 (21.97)    816.9729 (63.84)    562.7247 (46.87)    563.3615 (61.95)    338.1227 (75.39)         1;1   1.2240 (0.02)          5           1
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
============================= slowest 5 durations ==============================
8.36s call     tests/unit_tests/chat_models/test_azure_standard.py::TestOpenAIStandard::test_init_time
2.19s call     tests/unit_tests/chat_models/test_base_standard.py::TestOpenAIStandard::test_init_time
2.14s call     tests/unit_tests/chat_models/test_azure.py::test_initialize_azure_openai
0.88s call     tests/unit_tests/chat_models/test_responses_standard.py::TestOpenAIResponses::test_init_time
0.66s setup    tests/unit_tests/chat_models/test_azure_standard.py::TestOpenAIStandard::test_standard_params
=========================== short test summary info ============================
FAILED tests/unit_tests/chat_models/test_base.py::test__construct_lc_result_from_responses_api_web_search_response - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
FAILED tests/unit_tests/chat_models/test_base.py::test__construct_lc_result_from_responses_api_mixed_search_responses - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
= 2 failed, 192 passed, 37 deselected, 1 xfailed, 1 xpassed, 22 warnings in 29.72s =</pre>
</details>
<details>
<summary>python312Packages.ploomber-core</summary>
<pre>@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: ploomber_core
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
pytest flags: -m pytest -k not\ \(telemetry\)\ and\ not\ \(exceptions\)
ImportError while loading conftest '/private/tmp/nix-build-python3.12-ploomber-core-0.2.25.drv-0/source/tests/conftest.py'.
tests/conftest.py:5: in <module>
    import posthog
/nix/store/12jcvdmzhmchmmx9w7vwylmk8pspph16-python3.12-posthog-6.0.2/lib/python3.12/site-packages/posthog/__init__.py:3: in <module>
    from typing_extensions import Unpack
E   ModuleNotFoundError: No module named 'typing_extensions'</pre>
</details>
<details>
<summary>python313Packages.langchain-openai</summary>
<pre>Name (time in ms)          Min                 Max                Mean             StdDev              Median                IQR            Outliers      OPS            Rounds  Iterations
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_init_time          5.4650 (1.0)       58.9432 (1.0)       10.2148 (1.0)       7.3124 (1.0)        6.2665 (1.0)       8.6005 (1.47)         16;3  97.8969 (1.0)         124           1
test_init_time          5.5650 (1.02)      80.8487 (1.37)      11.9513 (1.17)     12.0525 (1.65)       7.3069 (1.17)      5.8696 (1.0)         10;11  83.6727 (0.85)        141           1
test_init_time        356.5691 (65.25)    397.0019 (6.74)     382.9902 (37.49)    16.3282 (2.23)     389.5464 (62.16)    21.3295 (3.63)          1;0   2.6110 (0.03)          5           1
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
============================= slowest 5 durations ==============================
3.71s call     tests/unit_tests/chat_models/test_azure_standard.py::TestOpenAIStandard::test_init_time
1.71s call     tests/unit_tests/chat_models/test_base_standard.py::TestOpenAIStandard::test_init_time
1.29s call     tests/unit_tests/chat_models/test_responses_standard.py::TestOpenAIResponses::test_init_time
0.32s call     tests/unit_tests/chat_models/test_azure.py::test_initialize_azure_openai
0.19s call     tests/unit_tests/llms/test_base.py::test_openai_fields_in_model_kwargs
=========================== short test summary info ============================
FAILED tests/unit_tests/chat_models/test_base.py::test__construct_lc_result_from_responses_api_web_search_response - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
FAILED tests/unit_tests/chat_models/test_base.py::test__construct_lc_result_from_responses_api_mixed_search_responses - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
= 2 failed, 192 passed, 37 deselected, 1 xfailed, 1 xpassed, 22 warnings in 11.04s =</pre>
</details>
<details>
<summary>python313Packages.ploomber-core</summary>
<pre>@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: ploomber_core
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
pytest flags: -m pytest -k not\ \(telemetry\)\ and\ not\ \(exceptions\)
ImportError while loading conftest '/private/tmp/nix-build-python3.13-ploomber-core-0.2.25.drv-0/source/tests/conftest.py'.
tests/conftest.py:5: in <module>
    import posthog
/nix/store/9gblvcb74z9kibd0kgcb9b1mkkvz09bp-python3.13-posthog-6.0.2/lib/python3.13/site-packages/posthog/__init__.py:3: in <module>
    from typing_extensions import Unpack
E   ModuleNotFoundError: No module named 'typing_extensions'</pre>
</details>
</details>

--- a/pkgs/development/python-modules/langchain-openai/default.nix
+++ b/pkgs/development/python-modules/langchain-openai/default.nix
@@ -32,14 +32,14 @@
 
 buildPythonPackage rec {
   pname = "langchain-openai";
-  version = "0.3.24";
+  version = "0.3.28";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
     tag = "langchain-openai==${version}";
-    hash = "sha256-eJqI7R1YzmVrZ+OoK2qtxkM2odpEDjszbBRD+2Gog9o=";
+    hash = "sha256-HpAdCHxmfGJcqXArvtlYagNuEBGBjrbICIwh9nI0qMQ=";
   };
 
   sourceRoot = "${src.name}/libs/partners/openai";

--- a/pkgs/development/python-modules/openai/default.nix
+++ b/pkgs/development/python-modules/openai/default.nix
@@ -3,7 +3,6 @@
   buildPythonPackage,
   fetchFromGitHub,
   pythonAtLeast,
-  pythonOlder,
 
   # build-system
   hatchling,
@@ -52,16 +51,14 @@
 
 buildPythonPackage rec {
   pname = "openai";
-  version = "1.91.0";
+  version = "1.97.0";
   pyproject = true;
-
-  disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "openai";
     repo = "openai-python";
     tag = "v${version}";
-    hash = "sha256-5thOFxXIStNowiEz9IacAkAC611zzXXs0ZB1tyuR+Go=";
+    hash = "sha256-q+GUEHducm71Zqh7ZfRF217awFKQIsOSEWoe04M3DFM=";
   };
 
   postPatch = ''substituteInPlace pyproject.toml --replace-fail "hatchling==1.26.3" "hatchling"'';
@@ -138,12 +135,12 @@ buildPythonPackage rec {
     "tests/api_resources"
   ];
 
-  meta = with lib; {
+  meta = {
     description = "Python client library for the OpenAI API";
     homepage = "https://github.com/openai/openai-python";
     changelog = "https://github.com/openai/openai-python/blob/v${version}/CHANGELOG.md";
-    license = licenses.mit;
-    maintainers = with maintainers; [ malo ];
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.malo ];
     mainProgram = "openai";
   };
 }

